File: core/src/main/java/org/toml/lang/lexer/TomlLexer.java
/*
 * Use of this source code is governed by the MIT license that can be
 * found in the LICENSE file.
 */

package org.toml.lang.lexer;

import consulo.language.lexer.FlexAdapter;

public class TomlLexer extends FlexAdapter {
    public TomlLexer() {
        super(new _TomlLexer(null));
    }
}

File: core/src/main/kotlin/org/toml/lang/lexer/TomlLexer.kt
/*
 * Use of this source code is governed by the MIT license that can be
 * found in the LICENSE file.
 */

package org.toml.lang.lexer

import consulo.language.lexer.FlexAdapter
import java.io.Reader

class TomlLexer : FlexAdapter(_TomlLexer(null as Reader?))
